### PR TITLE
Cleanup and move footer into separate component

### PR DESCRIPTION
**Pull request by SamSamskies**
Just doing a little cleanup to make it easier for new people to contribute.

### CODE REVIEW

**Comment by SamSamskies** [comment]
Looking for a volunteer to review this PR 😀

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+
+# WebStorm configs
+.idea
```
Comment (braidy-with-an-i) [inline .gitignore:33]: Github is complaining of no new line, might not be a bad idea to add in.

File: components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Icons by Flaticon
+      </a>
+    </footer>
+  );
+};
+
+export default Footer;
```
Comment (braidy-with-an-i) [inline components/Footer.js:27]: same question about adding a line at end of file here.

File: .gitignore (hunk 1 above)
Comment (SamSamskies) [reply .gitignore:33]: AFAIK it doesn't matter in this case, but it doesn't hurt. I'll update my IDE to auto add newlines on save. I have that setting on my work machine, but didn't have it set on my personal machine for some reason.

File: components/Footer.js (hunk 2 above)
Comment (SamSamskies) [reply components/Footer.js]: I updated the settings in my IDE to add newline on save.

**Comment by SamSamskies** [comment]
> Reviewed w/ Hannah, Emily, Braidy
> 
> _All good!_

Thanks! 🚀

**Pull request merged**